scripts/processors/.pylintrc

```diff
@@ -1,2 +1,2 @@
 [pylint.messages control]
-disable = unused-variable, line-too-long, invalid-name
+disable = unused-variable, line-too-long, invalid-name, redefined-outer-name, unused-argument, duplicate-code
```

Comment:
We are disabling lots of checks here. Wouldn't be better instead to update the code to respect the rules enforced by the linter?

Reply:
I think The line-too-long and duplicate-code checks should be ignored... Because there some commands in the scripts that violate the check rule!